### PR TITLE
Mapping with Rewards returns (with Okorin) newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
Much artist, many map.

### CODE REVIEW

File: news/2017-09-14-mapping-with-rewards-returns.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Mapping With Rewards Returns"
+date: 2017-09-14 18:00:00 +0000
+---
+
+New and improved, the open-form **Mapping With Rewards** mapping contests are back! Map a selection of [Featured Artist](https://osu.ppy.sh/beatmaps/artists) tracks for your shot at a variety of prizes, from osu!store credit, osu!supporter and even custom profile badges and titles!
```
Comment (MegaApplePi) [inline news/2017-09-14-mapping-with-rewards-returns.md]: serial comma, unless osu!supporter and the miscellaneous things _are_ a part of the osu!store credit 🤷‍♀️ 
`osu!supporter and even` ->
`osu!supporter, and even`

File: news/2017-09-14-mapping-with-rewards-returns.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+![](https://assets.ppy.sh/media/generic-header.png)
+
+**Mapping With Rewards** is an open-ended and judged mapping contest first floated in [September 2015](https://blog.ppy.sh/post/129936251068/mapping-with-rewards-sep-2015) with a simple premise: we offer a bounty for the best beatmap set of a [Featured Artist](https://osu.ppy.sh/beatmaps/artists) song, and a big timeframe to complete it in. Casual yet competitive at the same time!
+
+With its return, Mapping With Rewards is just as simple in concept as it was in the past, with the addition of expanding it to cover all gamemodes, and with a much-improved deadline process.
```
Comment (MegaApplePi) [inline news/2017-09-14-mapping-with-rewards-returns.md]: hmm.. is this a run-on sentence?

File: news/2017-09-14-mapping-with-rewards-returns.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/7/previews/71.mp3" type="audio/mpeg">
+</audio>
+
+> The cinematic climax of [Loki/Thaehan's](https://osu.ppy.sh/beatmaps/artists/7) *The Fall of a Hero*. High BPM symphonic metal with prominent electric guitar melodies and leads, set to muted but varied percussion.
++ [3:22 uncut, pre-timed .osz](https://assets.ppy.sh/artists/7/osz/Loki%20-%20A%20New%20King%20Is%20Born.osz)
+
+## Bounty
+The bounty is claimable **once per gamemode per song** - for the sake of simplicity osu!mania is treated as one game mode.
```
Comment (MegaApplePi) [inline news/2017-09-14-mapping-with-rewards-returns.md]: inconsistent spelling of term `gamemode`: `gamemode` is used 1 time, while `game mode` (with space) is used 7 times!

File: news/2017-09-14-mapping-with-rewards-returns.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+If you are judged to be the best map of the lot, here's what you can expect to win:
+
+- The beatmap will be bundled with the client for a limited period of time
```
Comment (MegaApplePi) [inline news/2017-09-14-mapping-with-rewards-returns.md]: `The beatmap` ->
`The beatmapset`?

File: news/2017-09-14-mapping-with-rewards-returns.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+- $50 osu!store credit
+- 6 months supporter tag
```
Comment (MegaApplePi) [inline news/2017-09-14-mapping-with-rewards-returns.md]: `supporter` ->
`osu!supporter`?

**Comment by Xinnoh** [comment]
Is it possible to use different mp3 names for short vs long version? Downloading both versions deletes one of the mp3s and combines sets, as the metadata is identical. This will cause anything mapped in the long version sections to be deleted, so probably worth fixing.

**Comment by Ephemeralis** [comment]
Applied changes from reviews - will look to get this out tonight after a few backend changes to the contest assets.

**Pull request merged**